docs/_deploying/heroku.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Deploying to Heroku
+order: 0
+slug: heroku
+lang: en
+layout: tutorial
+permalink: /deploying/heroku
+redirect_from:
+  - /deploying
+---
+# Deploying to Heroku
+
+<div class="section-content">
+This guide will walk you through preparing and deploying a Slack app using [Bolt for JavaScript](/bolt-js) and the [Heroku platform](https://heroku.com/). Along the way, we’ll download a Bolt Slack app, prepare it for Heroku, and deploy it.
+</div>
+
+When you’re finished, you’ll have this ⚡️[Getting Started with Heroku app](https://github.com/mwbrooks/bolt-js-getting-started-with-heroku-app) to run, modify, and make your own.
+
+---
+
+### Get a Bolt Slack app
+
+We'll continue from [Getting Started with Bolt for JavaScript][getting-started-guide] to learn how to prepare and deploy a Slack app to Heroku.
+
+Download the Getting Started app and change into its directory:
```

Comment:
If you use the suggestion above, I'd remove this line and separate out the `clone` command and the `cd` command (I think most people will already have an existing app).

The `clone` command can go after the first sentence and then under that, you could add a new sentence like "After you have a Bolt app, navigate to its directory:"

Reply:
Fixed in 3c9ab8f. Thanks for this suggestion. I think it flows much better than my original attempt and it addressed my concern that developers using an existing app may forget to change into the app's directory.